1-exercises/A-reading-list/script.js

```diff
@@ -1,26 +1,46 @@
 function readingList(books) {
-  // Write your code here...
+  let content = document.getElementById("content");
+  let bookList = document.createElement("ul");
+  content.appendChild(bookList);
```

Comment:
Good work. It is better practice to append the bookList to the content right at the end. Create the UL elemeent and add all the LI elements to this. Then finally, add the UL to the content element. The reason for this, is that you only update the browser DOM once. Every time it is updated, the web browser will refresh/rebuild the page content.